docs/attributes.md

```diff
@@ -0,0 +1,47 @@
+# Object Attributes (attrs)
+
+# This is still a work in progress (mistakes are common).
+
+## For objects that have associated events
+> Required in **bold**
+- duration: number; The number of users that can be admitted in the event associated to the event.
+- eventStartDelay: The number of minutes before the the event starts after the startAt field of the event.
+- eventDuration: number; The duration in minutes of the event associated to the object.
+- groupSize: number; The number of students that are required to be part of a group.
+- groupMax: number; The max amount of members a group should have to be created.
+- groupMin: number; The minimum amount of members a group should have to be created.
+- expectedXp: number; The amount of Xp an object gives.
+- baseXp: number; The starting point of the XP. It's used to calculate the xp per child object.
+
+## Some specific attributes for object types:
+- Exercise
+  - **language**: string; The programming language of the exercise, ex: 'go'
+  - start: type; Description
+  - delay: number; The amount of minutes that has to pass after the previous object before the current one is unlocked.
+  - special: boolean; True for a hackaton.
+  - startDay: number; The day relative to the beginning of the event associated to the parent event.
```

Comment:
this description seems a little bit complicated, maybe just "The day relative to the beginning of the parent event"

Reply:
Yes, that's better. Thank you